recipes/abacus/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "abacus" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/abacus-develop/archive/v{{ version }}.tar.gz
```

Comment:
This package seems to be revending libnpy. Revending and static linking are not allowed unless (as mentioned in the checklist) you include the license for the statically linked packages in the license section of this recipe (but you haven't). 

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=752531&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1483

Reply:
I checked https://github.com/llohse/libnpy and found it is a header-only package. I will add the license.